lib/lamby/templates/alb/_deploy

```diff
@@ -1,14 +1,24 @@
 #!/usr/bin/env bash
 set -e
 
+export DEPLOYMENT_NAME="myorg/APPNAMEHERE"
 export RAILS_ENV=${RAILS_ENV:="production"}
 export AWS_DEFAULT_REGION=${AWS_DEFAULT_REGION:=us-east-1}
-export CLOUDFORMATION_BUCKET=${CLOUDFORMATION_BUCKET:="lamby.cloudformation.$(whoami)"}
+export CLOUDFORMATION_BUCKET=${CLOUDFORMATION_BUCKET:="lamby.cloudformation.${DEPLOYMENT_NAME/\//-}"}
 
 # https://github.com/aws/aws-sam-cli/issues/2447
 export AWS_ACCOUNT_ID=$(aws sts get-caller-identity --query Account --output text)
-export IMAGE_REPOSITORY="${AWS_ACCOUNT_ID}.dkr.ecr.${AWS_DEFAULT_REGION}.amazonaws.com/myorg/APPNAMEHERE"
+export IMAGE_REPOSITORY="${AWS_ACCOUNT_ID}.dkr.ecr.${AWS_DEFAULT_REGION}.amazonaws.com/${DEPLOYMENT_NAME}"
 
+echo '== Creating ECR repository =='
+echo 'You can safely ignore any RepositoryAlreadyExistsException errors...'
+aws ecr create-repository \
+  --repository-name "$DEPLOYMENT_NAME" \
+  --image-tag-mutability MUTABLE \
+  --image-scanning-configuration scanOnPush=true \
+  --region "$AWS_DEFAULT_REGION" || true
+
+# Build our application code into a deployment folder
```

Comment:
Interesting. I guess there is an inflection point here. Makes these template files include more/all of the files here (https://github.com/customink/lamby-cookiecutter/tree/master/lambify) or simple files influenced by them. If the former, then this should bin in the bootstrap process. However, I think I'm going to lean to the later and merge this in.